app/components/donations/create-donation.js

```diff
@@ -1,9 +1,14 @@
 import Ember from 'ember';
 
-const { Component } = Ember;
+const { Component, computed, isEmpty } = Ember;
 
 export default Component.extend({
   classNames: ['create-donation'],
   presetAmounts: [10, 15, 25, 50],
-  amount: 15
+  amount: 10,
```

Comment:
Could we add `customAmount: null` here as well, so it is clear it's a property set on this component?